rumqttc/src/eventloop.rs

```diff
@@ -301,14 +306,14 @@ impl EventLoop {
         }
 
         // mqtt connection with timeout
-        time::timeout(Duration::from_secs(5), async {
+        time::timeout(self.timeout, async {
             network.connect(connect).await?;
             Ok::<_, ConnectionError>(())
         })
         .await??;
 
         // wait for 'timeout' time to validate connack
-        let packet = time::timeout(Duration::from_secs(5), async {
```

Comment:
Should  `timeout` for `connect` and `ack` should be controlled by 2 different params?

Reply:
No need if we are returning the same error